core/src/main/java/io/neow3j/transaction/TransactionBuilder.java

```diff
@@ -207,8 +201,21 @@ public TransactionBuilder additionalNetworkFee(long fee) {
     }
 
     /**
-     * Sets the script for this transaction. It defines the actions that this transaction will
-     * perform on the blockchain.
+     * Configures the transaction with an additional system fee.
+     * <p>
+     * The basic system fee required to send this transaction is added automatically.
+     *
```

Comment:
```suggestion
     * The basic system fee required to send this transaction is added automatically.
     * <p>
     * Use this, if you expect the transaction to consume more GAS because of chain state changes
     * happening between creating the transaction and actually sending it.
     *
```